requirements/edx/base.txt

```diff
@@ -291,7 +291,7 @@ django-fernet-fields==0.6
     #   -r requirements/edx/base.in
     #   edx-enterprise
     #   edxval
-django-filter==21.1
+django-filter==22.1
```

Comment:
No breaking change in new version https://github.com/carltongibson/django-filter/blob/main/CHANGES.rst#version-221-2022-6-17